### DISCUSSION TITLE
Mesh visualization doesn't match RViz

### DISCUSSION THREAD

**Discussion by samraise** [foxglove/community#997] [General]
Hi, I'm trying to visualize data coming off a mobile manipulator, but it looks like the meshes aren't loading properly (see image). What can I do to fix this?

![image](https://github.com/foxglove/community/assets/106622818/fdd53401-3bf3-4939-a6d3-a2a3c2a780b1)

**Comment by defunctzombie** [comment]
Please provide more information. For example how are you trying to reference the meshes? Are there any errors in the app? Do you expect these to load from URDF files? Marker messages?

**Comment by jtbandes** [comment]
Try the "Mesh up axis" setting: https://docs.foxglove.dev/docs/visualization/panels/3d/#scene
